torch/_export/wrappers.py

```diff
@@ -44,8 +44,8 @@ def export_tracepoint_functional(ctx, *args, **kwargs):
     unwrapped_kwargs = ctx.unwrap_tensors(kwargs)
 
     with ctx.redispatch_to_next():
-        out = _export_tracepoint(*unwrapped_args, **unwrapped_kwargs)
-        return ctx.wrap_tensors(out)
+        _export_tracepoint(*unwrapped_args, **unwrapped_kwargs)
+        return args
```

Comment:
nice catch (so far I guess I haven't seen any HOPs that return their inputs directly, so `ctx_wrap_tensors()` doesn't know that it might need to return inputs)